lua/entities/gmod_wire_expression2/core/console.lua

```diff
@@ -8,10 +8,12 @@ local function validConCmd(self, command)
 	local ply = self.player
 	if not ply:IsValid() then return false end
 	if ply:GetInfoNum("wire_expression2_concmd", 0) == 0 then return false end
-
+  	-- Validating the concmd length to ensure that it won't crash the server.
+  	if #command > 500000 then return false end
```

Comment:
This PR has whitespace issues... Why merge so quick @Nebual

Reply:
My bad 😅